.github/workflows/release-to-qa.yml

```diff
@@ -0,0 +1,19 @@
+name: Release to QA
+
+on:
+  schedule:
+    # Run on Tuesday-Sunday at 01:30 UTC, shortly before beginning of tester workday.
```

Comment:
This would be a great place to add a comment mentioning that it's possible to disable the workflow from the GitHub UI. Otherwise, someone won't realize that and will come here to comment out the schedule temporarily.